build.gradle.kts

```diff
@@ -257,5 +257,6 @@ tasks {
 
     runPluginVerifier {
         ideVersions.set(testIntellij.versions.versionList.get().split(',').map(String::trim).filter(String::isNotEmpty))
+        freeArgs = listOf("-mute", "TemplateWordInPluginId")
```

Comment:
The plugin verifier is complaining that the plugin id is named `com.github.bufbuild.intellij`, however we've already published under this id for some time so we should mute this warning.